protocol/database-scripting/logging.MD

```diff
@@ -0,0 +1,29 @@
+# Logging
+
+
+It is important that virtually every significant change to our database is tracked and auditable. 
+
+## PaperTrail
+In service of this, we use the [PaperTrail gem](https://github.com/airblade/paper_trail) to record the past versions/states of the records contained in most of our important database tables.
+
+In every rails console action, rake task, or other script, be sure to set a proper PaperTrail whodunnit field.
+
+To do this, execute the following command before any database transactions:
+```
```

Comment:
This could have `rb` syntax, since this code block is explicitly ran in irb, versus the regular terminal like the others.